test/server/http/admin_test.cc

```diff
@@ -191,7 +191,8 @@ TEST_P(AdminInstanceTest, EscapeHelpTextWithPunctuation) {
   Buffer::OwnedImpl response;
   EXPECT_EQ(Http::Code::OK, admin_.runCallback("/", header_map, response));
   Http::HeaderString& content_type = header_map.ContentType()->value();
-  EXPECT_TRUE(content_type.find("text/html")) << content_type.c_str();
+  EXPECT_THAT(content_type, testing::HasSubstr("text/html"))
```

Comment:
If you call `getString()` on content_type you should be able to use testing::HasSubstr.